frontend/ui/src/app/applications/application-version-detail-modal.component.html

```diff
@@ -0,0 +1,134 @@
+<div class="z-50 w-256 max-w-full max-h-full overflow-x-hidden overflow-y-auto">
+  <div class="relative w-full max-h-full">
+    <!-- Modal content -->
+    <div class="relative bg-white rounded-lg shadow-sm dark:bg-gray-700">
+      <!-- Modal header -->
+      <div class="flex items-center justify-between p-4 md:p-5 border-b border-gray-200 rounded-t dark:border-gray-600">
+        <h3 class="text-lg font-semibold text-gray-900 dark:text-white">
+          <span class="text-gray-500 dark:text-gray-400">{{ versionDetail().application.name }}</span>
+          {{ versionDetail().version.name }}
+        </h3>
+        <button
+          type="button"
+          class="text-gray-400 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm w-8 h-8 ms-auto inline-flex justify-center items-center dark:hover:bg-gray-600 dark:hover:text-white"
+          (click)="close()">
+          <svg class="w-3 h-3" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 14 14">
+            <path
+              stroke="currentColor"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="m1 1 6 6m0 0 6 6M7 7l6-6M7 7l-6 6" />
+          </svg>
+          <span class="sr-only">Close modal</span>
+        </button>
+      </div>
+
+      <!-- Modal body -->
+      <div class="p-4 md:p-5 space-y-4">
+        @if (versionDetail(); as detail) {
+          <div class="flex flex-wrap col-span-2 text-gray-500 dark:text-gray-400">
+            <!-- Link Tab (Common for both Docker and Kubernetes) -->
+            <input type="radio" id="tab-link" name="version-detail-tabs" class="hidden peer/link" checked />
+            <label
```

Comment:
I would change this to `tab-general` or something like that, as having a tab only for the link doesn't make so much sense IMO